website/manage.py

```diff
@@ -35,7 +35,7 @@
 # We haven't found anything helpful yet, so use development.
 if environment is None:
     try:
-        from . import configs.development.settings
+        import configs.development.settings
```

Comment:
I have a feeling this will only work if you're sitting in the "right" place, but I could be wrong (because there's a lot of complexity and something may have done the helpful thing somewhere in the stack). You can use `import .configs.development.settings` instead IIRC.

Reply:
Based on some amount of testing, the contrary appears to be true: `import configs.development.settings` appears to work from everywhere on the filesystem (including deep in the other app), but the dotted version doesn't appear to work anywhere, for either app.